### DISCUSSION TITLE
What is this used for?

### DISCUSSION THREAD

**Discussion by truegoodwill** [AlFasGD/VeryUnsafe#4] [Q&A]
Hi! Can you put an example of why and how one would use this, in the readme? Or just explain a little more here, please. I'd love to learn more about it.

**Comment by Rekkonnect** [comment]
The readme already explains some usage cases, with the notable one being promoting or demoting objects (changing their types to another type in the inheritance chain).

This library isn't intended to be anything special, it was made for fun after a discussion on Discord regarding type handles and seeing someone make a `void` instance creator function (also included in the readme). And as the name suggests, it's  very unsafe.
